src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Header {
-
+ get logo() {
+    return "Conduit logo";
+  }
 };
 
 class PageObject {
-
+  сonstructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, slug) {
```

Comment:
Call the super constructor using super() in the derived class constructor before accessing 'this'.

Reply:
done